CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Features
 
+* (tools) [#16550](https://github.com/cosmos/cosmos-sdk/pull/16550) Add COSMOVISOR_CUSTOM_PREUPGRADE to cosmovisor to execute custom pre-upgrade scripts.
```

Comment:
There is no changelog needed here. It should be under tools/cosmovisor/CHANGELOG.md